doc/xanadu_theme/static/xanadu.css_t

```diff
@@ -177,8 +177,8 @@ tt {
 }
 
 code, pre {
-  line-height: 23px;
-  margin: 20px 0;
+  line-height: 23px !important;
+  margin: 20px 0 !important;
```

Comment:
A recently released version of Pygments somehow is 'hijacking' our CSS settings here